### ISSUE TITLE
Is there any trained model checkpoint?

### ISSUE THREAD

**Issue by jooo5205** [fmiotello/dpai#1]
I  tried to start training so many times but nan value in compute loss comes out and training is not working well.

Is there any pre-trained model checkpoint I can get?

**Comment by fmiotello** [comment]
Hi!

I'm sorry to hear that. Unfortunately, I’m not maintaining this repository, and the code is a bit messy. Refactoring the code is on my to-do list, but I haven’t had the chance to get around to it yet.
Could the `NaN` values be related to [auraloss](https://github.com/csteinmetz1/auraloss), which I used for computing the loss function? Regarding pre-trained models, I don’t have any available. However, they wouldn’t make much sense in the context of the deep prior approach since a trained model is specifically used to reconstruct a single corrupted data sample and isn’t reusable for other data.

Apologies that I can’t be more helpful!
